<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelfInfoView</class>
 <widget class="QWidget" name="SelfInfoView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="logoutButton">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>187</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>47</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>照片</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="editInfoButton">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>127</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="securityButton">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>157</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>安全信息</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>221</width>
     <height>211</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="tel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>66</x>
       <y>119</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="departmentLabel">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>154</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>所在部门</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>83</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>出生年份</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>66</x>
       <y>18</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>18</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>姓名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>86</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>年龄</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="age">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>83</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>119</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>联系方式</string>
     </property>
    </widget>
    <widget class="QLabel" name="positionLabel">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>189</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>职位</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>53</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>性别</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bornYear">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>66</x>
       <y>83</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="sex">
     <property name="geometry">
      <rect>
       <x>66</x>
       <y>53</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>男</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>女</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="department">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>157</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>妇产科</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>儿科</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>内科</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>外科</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>神经科</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>皮肤性病科</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="position">
     <property name="geometry">
      <rect>
       <x>69</x>
       <y>187</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>主任</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>副主任</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>主治医师</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>医师</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>助理医师</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>实习生</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLineEdit" name="tel_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>153</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>117</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>出生年份</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>44</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>44</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>姓名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>129</x>
       <y>117</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>年龄</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="age_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>159</x>
       <y>117</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>153</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>联系方式</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>79</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>性别</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bornYear_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>117</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="sex_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>男</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>女</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editInfoButton</tabstop>
  <tabstop>securityButton</tabstop>
  <tabstop>logoutButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
